src/main/java/hudson/plugins/analysis/views/AttributeDetail.java

```diff
@@ -42,5 +43,28 @@ public AttributeDetail(final AbstractBuild<?, ?> owner, final DetailFactory deta
     public String getDisplayName() {
         return attributeName;
     }
+
+    /**
+     * Creates a new instance of {@link AttributeDetail}.
+     *
+     * @param owner
+     *            current build as owner of this action.
+     * @param detailFactory
+     *            factory to create detail objects with
+     * @param annotations
+     *            the module to show the details for
+     * @param header
+     *            header to be shown on detail page
+     * @param defaultEncoding
+     *            the default encoding to be used when reading and parsing files
+     * @param name
+     *            name of the attribute shown in the bread crumb
+     * @deprecated use {@link #AttributeDetail(Run, DetailFactory, Collection, String, String, String)} instead
+     */
+    @Deprecated
+    public AttributeDetail(final AbstractBuild<?, ?> owner, final DetailFactory detailFactory, final Collection<FileAnnotation> annotations, final String defaultEncoding, final String header, final String name) {
+        super(owner, detailFactory, annotations, defaultEncoding, header, Hierarchy.PROJECT);
```

Comment:
`this((Run<?,?>)owner, ...)` would be simpler and safer for further class evolution down the line.